File: modul_praktikum_python/contoh_code/python/tunjangan_anak.py
"""Hitung tunjangan berdasarkan gaji kotor dan jumlah anak."""


def main() -> None:
    persen_tunjangan = 0.2
    gaji_kotor = float(input("Gaji Kotor ?"))
    jumlah_anak = int(input("Jumlah Anak ?"))
    if jumlah_anak > 2:
        persen_tunjangan = 0.3
    tunjangan = persen_tunjangan * gaji_kotor
    print(f"Besar Tunjangan = Rp {tunjangan:10.2f}")
    input("Tekan Enter untuk melanjutkan...")


if __name__ == "__main__":
    main()

File: modul_praktikum_python/contoh_code/pascal/tunjangan_anak.pas
uses crt; 
var 
  JumlahAnak : integer; 
  GajiKotor,Tunjangan,PersenTunjangan : real ; 
begin 
  PersenTunjangan:=0.2; 
  write(‘Gaji Kotor ?’);readln(GajiKotor); 
  write(‘Jumlah Anak ?’);readln(JumlahAnak); 
  if JumlahAnak>2 then  
     PersenTunjangan:=0.3; 
  Tunjangan := PersenTunjangan*GajiKotor; 
  writeln(‘Besar Tunjangan = Rp ’,Tunjangan:10:2); 
  readkey;   
end.
